.github/workflows/crucible-ci.yaml

```diff
@@ -1,9 +1,7 @@
 name: crucible-ci
 
 on:
-  # run on push or pull request events for the main branch only
-  push:
```

Comment:
this won't run ci after merging... would not catch problems after merging the code... depending on what our "integration" test does, this may not be relevant

Reply:
This change is really just making it the same as every other repo at the moment.  I do think that we might want to explore this further, however, given our current merge rules I'm not sure that doing a post-merge test accomplishes anything because its really just a repeat of the PR run (since we force PRs to be updated before merging is allowed and we require CI to pass, there is never upstream code that hasn't been tested with the new PR code prior to merging).

That said, I'm not really opposed to running more CI, but I would like to see it only happen on the upstream repositories.  I don't want to be running CI on everyone's forks which is what was happening here.